OracleDatabase/SingleInstance/FAQ.md

```diff
@@ -46,3 +46,15 @@ As you don't have execution rights to it, however, you get the error `Operation
 Run the container with `-v /dev/shm --tmpfs /dev/shm:rw,exec,size=<yoursize>` instead, the important part being the `exec` in `--tmpfs /dev/shm:rw,exec,size=<yoursize>`.
 Also make sure you assign an appropriate size as the default Docker uses is only 64MB. Assigning 1GB or  more is recommended.
 
+## ORA-12637: Packet receive failed
+When initially connecting to your database via `sqlplus` the connection may appear to hang and timeout after a few minutes with:
+```
+ERROR:
+ORA-12637: Packet receive failed
+
+
+Enter user-name:
```

Comment:
No need to show the 'Enter user-name', particularly as other affected tools & drivers don't have this problem